### ISSUE TITLE
Triggers can't be simultaneously activated if they have a shared parent with a rigidbody

### ISSUE THREAD

**Issue by funcmathias** [Facepunch/sbox-issues#6182]
### Describe the bug

When two triggers share a parent with a rigidbody they can no longer be activated simultaneously, so if they overlap you have to first exit one to activate the other.
Video here demonstrating with and without a rigidbody:

https://github.com/user-attachments/assets/8b3b0761-84eb-4e71-8003-e8722e343342

### To Reproduce

Have two game objects with a collider set to trigger and a script calling OnTriggerEnter and Exit, and make their colliders overlap.
Add both as children to the same parent and test, both should trigger fine.
Add a rigidbody component to the shared parent (and probably turn off gravity and motion).
Test again and now only one of the triggers can be active at a time, included video demonstrates this as well.

### Expected behavior

Both triggers should work the same as before the rigidbody is added to the shared parent.

**Comment by aylaylay** [comment]
It's hard to say if this should be the desired behaviour or not. When you add a rigidbody it collects all the colliders it can finds so it acts as a single body with multiple shapes.

**Comment by funcmathias** [comment]
Yeah I assumed something like that was the reason for it ending up like that, but I don't think it makes sense for the triggers to be included as part of the rigidbody as they should have no effect on the physics behavior. Before I found out exactly what was happening the triggers just seemed unreliable, and I can't see how a trigger not triggering would be desired (unless manually disabled of course).

The way I discovered this behavior is I wanted a player controller with a big trigger around it to know which objects to push with a shockwave, and a smaller trigger on the player to check for objects to grab, and in this setup the grab trigger would not function. Currently as a workaround I've had to replace the big trigger with a distance check for all objects.

Also I checked how Unity treats it and it seems to line up with what I expect at least, for what it's worth.

https://github.com/user-attachments/assets/ef3a304e-d5c7-4ff4-882f-6d942f9f1c7d
